modules/assim.sequential/R/Analysis_sda.R

```diff
@@ -123,11 +123,10 @@ GEF<-function(setting,Forecast,Observed, H, extraArg, nitr=50000, nburnin=10000,
   #Forecast inputs 
   Q <- Forecast$Q # process error
   X <- Forecast$X # states 
-  Pf = cov(X) # Cov Forecast - This is used as an initial condition
-
+  Pf = cov(X) # Cov Forecast - Goes into tobit2space as initial condition but is re-estimated in tobit space
   mu.f <- colMeans(X) #mean Forecast - This is used as an initial condition
   #Observed inputs
-  R <- Observed$R
+  R <- solve(Observed$R) #putting solve() here so if not invertible error is before compiling tobit2space
```

Comment:
Should this be in a `try` statement or similar in case the matrix is not invertible and it crashes the workflow?

Reply:
It's sort of a big problem if the R is not invertible right? It should be checking in data preprocessing but if it isn't it'll throw an error here. But, I could write out a warning that tells the user to go back to data processing?